Print_A_Bot/djwifi/templates/djwifi/configure.html

```diff
@@ -0,0 +1,246 @@
+{% extends 'djwifi/base.html' %}
+
+{% load i18n %}
+
+{% block meta_title %}{% trans 'Configure Wifi' %}{% endblock %}
+
+{% block content %}
+    {% if configured %}
+        <div id="wifi-current-wrapper"></div>
+        <div id="wifi-wrapper">
+            <div class="text-center">
+                <i class="fa fa-spinner fa-spin fa-5x"></i>
+            </div>
+        </div>
+    {% else %}
+        <h1 class="text-center">{% trans 'Wifi not available.' %}</h1>
+    {% endif %}
+{% endblock %}
+
+
+{% block extra_scripts %}
+<script id="template-wifi-current" type="text/x-handlebars-template">
+    <div class="panel panel-default text-center">
+        <div class="panel-heading">
+            <h3 class="panel-title">
+                {% trans 'Connected to' %} {% verbatim %}{{ essid }}{% endverbatim %}
+            </h3>
+        </div>
+        <div class="panel-body">
+            {% verbatim %}{{ quality }}%{% endverbatim %} {% trans 'signal' %}
```

Comment:
verbatim tag prevents django from attempting to render the code inside

handlebars and django use a very similar template syntax but handlebars being javascript runs inside the browser